monero/wallet_client.go

```diff
@@ -351,9 +443,9 @@ func createWalletRPCService(conf *WalletClientConf) (string, *os.Process, error)
 func getMoneroWalletRPCBin() (string, error) {
 	execName := "monero-wallet-rpc"
 	priorityPath := path.Join("monero-bin", execName)
-	path, err := exec.LookPath(priorityPath)
+	execPath, err := exec.LookPath(priorityPath)
```

Comment:
Variable name conflicted with import name. My IDE warns about it, but I don't think our linter does. If we find the knob to enabled that check, I think it would be good.